src/main/kotlin/g0201_0300/s0202_happy_number/readme.md

```diff
@@ -0,0 +1,27 @@
+201\. Bitwise AND of Numbers Range
```

Comment:
Please update generatemd tool to the latest version and change readme to task 202.